### ISSUE TITLE
Can't get extended status to work on Windows

### ISSUE THREAD

**Issue by ka-may** [creativeprojects/resticprofile#46]
Hey!
Can't get extended status to work on Windows.

Here's an example of my configuration:

```
> profile:
>     repository: "gs:bareos1:/A3"
>     password-file: "password.txt"
>     status-file: C:\resticprofile\stat4.json
>     backup:
>         extended-status: true
>         source: C:\resticprofile
>         schedule:
>         - "Mon..Fri *:00,25,28,29" # every 15 minutes on weekdays
>         schedule-permission: user
```

        
I run resticprofile with the command:
`C:\Windows\system32\resticprofile.exe --no-ansi --config c:\resticprofile\profiles.yaml --name profile backup`

But such values ​​as the volume of files or the number of folders still do not change and remain equal to 0.

The backup succeeds - you can see this by the exit code

`{"profiles":{"profile":{"backup":{"success":true,"time":"2021-04-20T12:46:09.321827+03:00","error":"","stderr":"","duration":11,"files_new":0,"files_changed":0,"files_unmodified":0,"dirs_new":0,"dirs_changed":0,"dirs_unmodified":0,"files_total":0,"bytes_added":0,"bytes_total":0}}}}`

        
My version resticprofile 0.13.1

**Comment by creativeprojects** [comment]
Thanks for your report, I'll see if I can reproduce it

**Comment by ka-may** [comment]
> Спасибо за отчет, посмотрю, смогу ли я его воспроизвести

Спасибо- буду ждать!

**Comment by ka-may** [comment]
> Thanks for your report, I'll see if I can reproduce it
Forgot to write:

After adding the **extended-status: true** key, the command line output has changed and it looks like this

←[2K{"message_type":"summary","files_new":0,"files_changed":1,"files_unmodified":20,"dirs_new":0,"dirs_changed":2,"dirs_unmodified":2,"data_blobs":1,"tree_blobs":3,"data_added":5004,"total_files_processed":21,"total_bytes_processed":636644022,"total_duration":4.0247055,"snapshot_id":"d0f6f86d"}

**Comment by creativeprojects** [comment]
Which terminal are you using when running the resticprofile command?
Also are you using Cmd, Powershell or Bash for Windows?

**Comment by creativeprojects** [comment]
I tried the new Windows Terminal and I found the funny character `\33[2K` being at the start of each output line, which is the issue.

It's actually a well known issue of `restic`: https://github.com/restic/restic/issues/3111

I'll try a workaround 😄

**Comment by ka-may** [comment]
> Какой терминал вы используете при запуске команды resticprofile?
> Вы также используете Cmd, Powershell или Bash для Windows?

Я использую CMD

**Comment by creativeprojects** [comment]
Your translator doesn't work well by the way, some of your messages (but not all) are showing in Russian 😆 

<img width="580" alt="image" src="https://user-images.githubusercontent.com/24314560/115434646-d6a88480-a200-11eb-9649-d194262f18a5.png">

**Comment by ka-may** [comment]
> Your translator doesn't work well by the way, some of your messages (but not all) are showing in Russian 😆
> 
> <img alt="image" width="580" src="https://user-images.githubusercontent.com/24314560/115434646-d6a88480-a200-11eb-9649-d194262f18a5.png">

Unfortunately, this is not a translator, but I wrote it right away in Russian. :)

When you run the Powershell chz, nothing changes to the status file either, the output is:


```
PS C:\Users\shaptala.d> C:\Windows\system32\resticprofile.exe --no-ansi --config c:\resticprofile\profiles.yaml --name profile backup
2021/04/20 19:49:40 profile 'profile': starting 'backup'
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":1,"total_bytes":284}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":2,"total_bytes":970}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":3,"total_bytes":1250}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":4,"total_bytes":3584}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":5,"total_bytes":3613}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":6,"total_bytes":3832}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":7,"total_bytes":3842}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":8,"total_bytes":4324}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":9,"total_bytes":4926}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":10,"total_bytes":5436}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":11,"total_bytes":5945}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":12,"total_bytes":6306}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":13,"total_bytes":154921}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":14,"total_bytes":875817}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":15,"total_bytes":987377}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":16,"total_bytes":8027889}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":17,"total_bytes":15067889}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":18,"total_bytes":15082507}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":19,"total_bytes":23309547}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":20,"total_bytes":23320935}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":21,"total_bytes":636644105}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":21,"total_bytes":636644105}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":21,"total_bytes":636644105}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":21,"total_bytes":636644105}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":21,"total_bytes":636644105}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":21,"files_done":1,"total_bytes":636644105}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":21,"files_done":1,"total_bytes":636644105}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0,"total_files":21,"files_done":1,"total_bytes":636644105}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":4.460891065660617e-7,"total_files":21,"files_done":1,"total_bytes":636644105,"bytes_done":284}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":4.460891065660617e-7,"total_files":21,"files_done":2,"total_bytes":636644105,"bytes_done":284}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":4.460891065660617e-7,"total_files":21,"files_done":2,"total_bytes":636644105,"bytes_done":284}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":4.460891065660617e-7,"total_files":21,"files_done":2,"total_bytes":636644105,"bytes_done":284}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.0000015236142020038025,"total_files":21,"files_done":2,"total_bytes":636644105,"bytes_done":970}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.0000015236142020038025,"total_files":21,"files_done":3,"total_bytes":636644105,"bytes_done":970}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.0000015236142020038025,"total_files":21,"files_done":3,"total_bytes":636644105,"bytes_done":970}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.0000015236142020038025,"total_files":21,"files_done":3,"total_bytes":636644105,"bytes_done":970}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000001963420363406962,"total_files":21,"files_done":3,"total_bytes":636644105,"bytes_done":1250}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000001963420363406962,"total_files":21,"files_done":4,"total_bytes":636644105,"bytes_done":1250}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000001963420363406962,"total_files":21,"files_done":4,"total_bytes":636644105,"bytes_done":1250}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000001963420363406962,"total_files":21,"files_done":4,"total_bytes":636644105,"bytes_done":1250}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.0000056295188659604415,"total_files":21,"files_done":4,"total_bytes":636644105,"bytes_done":3584}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.0000056295188659604415,"total_files":21,"files_done":4,"total_bytes":636644105,"bytes_done":3584,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.0000056295188659604415,"total_files":21,"files_done":5,"total_bytes":636644105,"bytes_done":3584,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.0000056295188659604415,"total_files":21,"files_done":5,"total_bytes":636644105,"bytes_done":3584,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.0000056295188659604415,"total_files":21,"files_done":5,"total_bytes":636644105,"bytes_done":3584,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000005973510113629341,"total_files":21,"files_done":5,"total_bytes":636644105,"bytes_done":3803,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000005973510113629341,"total_files":21,"files_done":6,"total_bytes":636644105,"bytes_done":3803,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000005973510113629341,"total_files":21,"files_done":6,"total_bytes":636644105,"bytes_done":3803,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000005973510113629341,"total_files":21,"files_done":6,"total_bytes":636644105,"bytes_done":3803,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000005989217476536597,"total_files":21,"files_done":6,"total_bytes":636644105,"bytes_done":3813,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000005989217476536597,"total_files":21,"files_done":7,"total_bytes":636644105,"bytes_done":3813,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000005989217476536597,"total_files":21,"files_done":7,"total_bytes":636644105,"bytes_done":3813,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000005989217476536597,"total_files":21,"files_done":7,"total_bytes":636644105,"bytes_done":3813,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000006746312368666321,"total_files":21,"files_done":7,"total_bytes":636644105,"bytes_done":4295,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000006746312368666321,"total_files":21,"files_done":8,"total_bytes":636644105,"bytes_done":4295,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000006746312368666321,"total_files":21,"files_done":8,"total_bytes":636644105,"bytes_done":4295,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000006746312368666321,"total_files":21,"files_done":8,"total_bytes":636644105,"bytes_done":4295,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000007691895615683114,"total_files":21,"files_done":8,"total_bytes":636644105,"bytes_done":4897,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000007737446968114155,"total_files":21,"files_done":8,"total_bytes":636644105,"bytes_done":4926,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000007737446968114155,"total_files":21,"files_done":9,"total_bytes":636644105,"bytes_done":4926,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000007737446968114155,"total_files":21,"files_done":9,"total_bytes":636644105,"bytes_done":4926,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000007737446968114155,"total_files":21,"files_done":9,"total_bytes":636644105,"bytes_done":4926,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000008538522476384196,"total_files":21,"files_done":9,"total_bytes":636644105,"bytes_done":5436,"current_files":["/C/resticprofile/bat.bat"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000008538522476384196,"total_files":21,"files_done":9,"total_bytes":636644105,"bytes_done":5436,"current_files":["/C/resticprofile/bat.bat","/C/resticprofile/stat4.json"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000008538522476384196,"total_files":21,"files_done":10,"total_bytes":636644105,"bytes_done":5436,"current_files":["/C/resticprofile/bat.bat","/C/resticprofile/stat4.json"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000008538522476384196,"total_files":21,"files_done":10,"total_bytes":636644105,"bytes_done":5436,"current_files":["/C/resticprofile/bat.bat","/C/resticprofile/stat4.json"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000008538522476384196,"total_files":21,"files_done":10,"total_bytes":636644105,"bytes_done":5436,"current_files":["/C/resticprofile/bat.bat","/C/resticprofile/stat4.json"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.000009105558277336127,"total_files":21,"files_done":10,"total_bytes":636644105,"bytes_done":5797,"current_files":["/C/resticprofile/bat.bat","/C/resticprofile/stat4.json"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.00000990506304931544,"total_files":21,"files_done":10,"total_bytes":636644105,"bytes_done":6306,"current_files":["/C/resticprofile/bat.bat","/C/resticprofile/stat4.json"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.00000990506304931544,"total_files":21,"files_done":11,"total_bytes":636644105,"bytes_done":6306,"current_files":["/C/resticprofile/bat.bat","/C/resticprofile/stat4.json"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.00000990506304931544,"total_files":21,"files_done":11,"total_bytes":636644105,"bytes_done":6306,"current_files":["/C/resticprofile/stat4.json"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.00000990506304931544,"total_files":21,"files_done":11,"total_bytes":636644105,"bytes_done":6306,"current_files":["/C/resticprofile/stat4.json"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.00000990506304931544,"total_files":21,"files_done":12,"total_bytes":636644105,"bytes_done":6306,"current_files":["/C/resticprofile/stat4.json"]}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.00000990506304931544,"total_files":21,"files_done":12,"total_bytes":636644105,"bytes_done":6306}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.00000990506304931544,"total_files":21,"files_done":12,"total_bytes":636644105,"bytes_done":6306}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.00000990506304931544,"total_files":21,"files_done":13,"total_bytes":636644105,"bytes_done":6306}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.00000990506304931544,"total_files":21,"files_done":13,"total_bytes":636644105,"bytes_done":6306}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.00000990506304931544,"total_files":21,"files_done":13,"total_bytes":636644105,"bytes_done":6306}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.00024334003689549595,"total_files":21,"files_done":13,"total_bytes":636644105,"bytes_done":154921}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.00024334003689549595,"total_files":21,"files_done":14,"total_bytes":636644105,"bytes_done":154921}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.00024334003689549595,"total_files":21,"files_done":14,"total_bytes":636644105,"bytes_done":154921}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.00024334003689549595,"total_files":21,"files_done":14,"total_bytes":636644105,"bytes_done":154921}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.001375677545934396,"total_files":21,"files_done":14,"total_bytes":636644105,"bytes_done":875817}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.001375677545934396,"total_files":21,"files_done":15,"total_bytes":636644105,"bytes_done":875817}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.001375677545934396,"total_files":21,"files_done":15,"total_bytes":636644105,"bytes_done":875817}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.001375677545934396,"total_files":21,"files_done":15,"total_bytes":636644105,"bytes_done":875817}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.0015509088865277407,"total_files":21,"files_done":15,"total_bytes":636644105,"bytes_done":987377}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.0015509088865277407,"total_files":21,"files_done":16,"total_bytes":636644105,"bytes_done":987377}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.0015509088865277407,"total_files":21,"files_done":16,"total_bytes":636644105,"bytes_done":987377}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.0015509088865277407,"total_files":21,"files_done":16,"total_bytes":636644105,"bytes_done":987377}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.012609696590216601,"total_files":21,"files_done":16,"total_bytes":636644105,"bytes_done":8027889}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.012609696590216601,"total_files":21,"files_done":17,"total_bytes":636644105,"bytes_done":8027889}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.012609696590216601,"total_files":21,"files_done":17,"total_bytes":636644105,"bytes_done":8027889}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.012609696590216601,"total_files":21,"files_done":17,"total_bytes":636644105,"bytes_done":8027889}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.023667680076924612,"total_files":21,"files_done":17,"total_bytes":636644105,"bytes_done":15067889}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.023667680076924612,"total_files":21,"files_done":18,"total_bytes":636644105,"bytes_done":15067889}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.023667680076924612,"total_files":21,"files_done":18,"total_bytes":636644105,"bytes_done":15067889}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.023667680076924612,"total_files":21,"files_done":18,"total_bytes":636644105,"bytes_done":15067889}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.023690641100022437,"total_files":21,"files_done":18,"total_bytes":636644105,"bytes_done":15082507}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.023690641100022437,"total_files":21,"files_done":19,"total_bytes":636644105,"bytes_done":15082507}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.023690641100022437,"total_files":21,"files_done":19,"total_bytes":636644105,"bytes_done":15082507}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.023690641100022437,"total_files":21,"files_done":19,"total_bytes":636644105,"bytes_done":15082507}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.03661315139327333,"total_files":21,"files_done":19,"total_bytes":636644105,"bytes_done":23309547}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.03661315139327333,"total_files":21,"files_done":20,"total_bytes":636644105,"bytes_done":23309547}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.03661315139327333,"total_files":21,"files_done":20,"total_bytes":636644105,"bytes_done":23309547}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.03661315139327333,"total_files":21,"files_done":20,"total_bytes":636644105,"bytes_done":23309547}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.03663103893815211,"total_files":21,"files_done":20,"total_bytes":636644105,"bytes_done":23320935}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.03663103893815211,"total_files":21,"files_done":21,"total_bytes":636644105,"bytes_done":23320935}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.03663103893815211,"total_files":21,"files_done":21,"total_bytes":636644105,"bytes_done":23320935}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":0.03663103893815211,"total_files":21,"files_done":21,"total_bytes":636644105,"bytes_done":23320935}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":1,"total_files":21,"files_done":21,"total_bytes":636644105,"bytes_done":636644105}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":1,"total_files":21,"files_done":21,"total_bytes":636644105,"bytes_done":636644105}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":1,"total_files":21,"files_done":21,"total_bytes":636644105,"bytes_done":636644105}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":1,"total_files":21,"files_done":21,"total_bytes":636644105,"bytes_done":636644105}
←[2K{"message_type":"status","seconds_elapsed":3,"percent_done":1,"total_files":21,"files_done":21,"total_bytes":636644105,"bytes_done":636644105}
←[2K{"message_type":"status","seconds_elapsed":4,"percent_done":1,"total_files":21,"files_done":21,"total_bytes":636644105,"bytes_done":636644105}
←[2K{"message_type":"summary","files_new":0,"files_changed":2,"files_unmodified":19,"dirs_new":0,"dirs_changed":2,"dirs_unmodified":1,"data_blobs":2,"tree_blobs":3,"data_added":4974,"total_files_processed":21,"total_bytes_processed":636644105,"total_duration":4.107006,"snapshot_id":"01ce89d1"}

2021/04/20 19:49:45 profile 'profile': finished 'backup'
```

**Comment by creativeprojects** [comment]
Thanks 😄 

I made a quick workaround and tested it successfully on Windows.
I'm going to test it on the other platforms to make sure it still works and I'll publish it shortly 👍🏻

**Comment by ka-may** [comment]
> Спасибо 😄
> 
> Я нашел быстрое решение и успешно протестировал его в Windows.
> Я собираюсь протестировать его на других платформах, чтобы убедиться, что он все еще работает, и вскоре опубликую его 👍🏻

I will wait, thanks!

**Comment by creativeprojects** [comment]
Fix is available on release v0.13.2:
https://github.com/creativeprojects/resticprofile/releases/tag/v0.13.2
